spec/models/account_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Account, type: :model do
+  let!(:account) { FactoryBot.create(:account) }
+  let(:edrpou) { account.edrpou }
+  let(:iban) { account.iban }
+  let(:purpose) { account.purpose }
+
+  context "when valid Factory" do
+    it "has a valid factory" do
+      expect(account).to be_valid
+    end
+  end
+
+  describe 'validations' do
+    context 'when edrpou' do
+      it 'require a edrpou' do
+        account.edrpou = edrpou
```

Comment:
why do do you do this, this is the same that account.edprou = account.edprou, and since you created it with factory in line 4, that is already valid

Reply:
done